test/uk/gov/hmrc/emailaddress/EmailAddressGenerators.scala

```diff
@@ -16,21 +16,38 @@
 
 package uk.gov.hmrc.emailaddress
 
-import org.scalacheck.Gen._
+import scala.collection.mutable.ListBuffer
+import scala.util.Random
 
-trait EmailAddressGenerators {
+object EmailAddressGenerators {
 
-  val nonEmptyString = nonEmptyListOf(alphaChar).map(_.mkString)
+  lazy val random: Random = new Random()
 
-  val validMailbox = nonEmptyString
+  def randomEmailAddresses(): List[String] = {
+    var res = ListBuffer[String]()
```

Comment:
This can be done using a for yield instead of a variable

Reply:
You should look at scalacheck generators for property based testing

Reply:
I try to test with these scalacheck generators but end with a lot of errors if hard predictable test cases, now you can predict at least size of string and characters in this string

If you want we can revert this class and do a pair session to see how to improve this method from scalacheck, if this is the right way to do random data for test's